x-pack/filebeat/include/list.go

```diff
@@ -10,6 +10,7 @@ import (
 	// Import packages that need to register themselves.
 	_ "github.com/elastic/beats/x-pack/filebeat/input/netflow"
 	_ "github.com/elastic/beats/x-pack/filebeat/module/iptables"
+        _ "github.com/elastic/beats/x-pack/filebeat/module/netflow"
 	_ "github.com/elastic/beats/x-pack/filebeat/module/kubernetes"
```

Comment:
You mean "Add Filebeat netflow module"? :-) Also, is tabs vs spaces an issue with Go?

Reply:
Sorry, was looking at a single commit, never mind ;-)